fractal/cmd/_aux_task_caching.py

```diff
@@ -20,7 +20,16 @@ async def _fetch_task_list(client: AuthClient) -> _TaskList:
     """
     res = await client.get(f"{settings.BASE_URL}/task/")
     task_list = check_response(res, expected_status_code=200)
-    return task_list
+    return [
+        dict(
```

Comment:
What about `dict(include=...)`?
(https://docs.pydantic.dev/latest/usage/exporting_models/#advanced-include-and-exclude)

Reply:
My bad, this was for Pydantic objects but we only use dictionaries in this function.